### DISCUSSION TITLE
Welcome to solis_modbus Discussions!

### DISCUSSION THREAD

**Discussion by Pho3niX90** [Pho3niX90/solis_modbus#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by UDicke** [comment]
I'm now a HA user for about 1.5 years and am very satisfied with the solutions available in the HA environment. I have a SAJ inverter and use evcc for excess charging. Now I want to help my brother-in-law to integrate his Solis S6-EH3P into HA locally using this integration without giving up the cloud connection. He has a Waveshare module, but unfortunately, I can't find the installation instructions. I saw them somewhere, but I can't remember where, and I need help! :-)
Thank you for providing this integration and for you help!

**Reply by 0rangutan** [reply]
Here you go: https://github.com/alienatedsec/solis-ha-modbus-cloud

**Reply by UDicke** [reply]
Thank you! This will help a lot. 👍

**Reply by julmnk** [reply]
btw this [solis-ha-modbus-cloud](https://github.com/alienatedsec/solis-ha-modbus-cloud) integration does really need two Waveshare adapters and wiring data Logger which in Solis S6-EH3P probably will be USB and not 4 pins like in the link if RS485 port is not used by something else! My S6-EH3P15K02-NV-L has own Meter port and a separate RS485 which I used with Waveshare adapter from HA ModBus integration.
